### ISSUE TITLE
csproj parsing error

### ISSUE THREAD

**Issue by kresli** [fernandoescolar/vscode-solution-explorer#118]
on any csproj file modification by extension the xml became invalid if contain `&amp`. 
from 
`<Content Include="help\Content\Boards &amp; Widgets.htm" />`
to 
`<Content Include="help\Content\Boards & Widgets.htm" />`

and error
```
error: Can not create file: Error: Invalid character in entity name
Line: 155
Column: 44
Char:  
```

**Comment by fernandoescolar** [comment]
Special characters as '&', '>' or '>' in file names are not allowed in Visual Studio Enterprise, Professional or Community:

![image](https://user-images.githubusercontent.com/3491424/112726187-6ae23d00-8f1c-11eb-8ffe-98d44f327420.png)

Anyway I have added a fix in the XML parser to save some special characters encoded:

```
& -> &amp;
< -> &lt;
> -> &gt;
" -> &quot;
' -> &apos;
```

You will find it in the next version.

**Comment by kresli** [comment]
Yeah, not sure if this is from legacy files.
Unfortunately the fix doesn't work.
![image](https://user-images.githubusercontent.com/3826097/113699146-1c2b6480-96cd-11eb-826a-0330047e372d.png)

Before create a new file
![image](https://user-images.githubusercontent.com/3826097/113699229-306f6180-96cd-11eb-8171-778296d3f121.png)
After Create a new file
![image](https://user-images.githubusercontent.com/3826097/113699305-48df7c00-96cd-11eb-8048-24cf4d9134b0.png)

Basically we dont want to replace `&amp;` by `&`

**Comment by kresli** [comment]
Ok, I see you reverted the change.

**Comment by kresli** [comment]
#166
